packages/fast-components-react-base/src/label/label.props.ts

```diff
@@ -0,0 +1,26 @@
+import * as React from "react";
+import { ILabelClassNameContract, IManagedClasses } from "@microsoft/fast-components-class-name-contracts-base";
+
+export enum LabelTag {
+    div = "div",
```

Comment:
I wonder if we want to expose label as a <p> or <span> now that we have the typography component. There doesn't seem to be any added benefit by exposing it as such aside from added complexity within the design system.

Reply:
Talked this over offline - will only expose `label` and 'legend`